### PR TITLE
pkg_deb supports license attribute

### LINKED ISSUE #650: pkg_deb license attribute is not supported
Hi, before I have a look if I could help to contribute support for the license attribute: is there a known reason why `pkg_deb` does not already support the `license` attribute? Can this already be set another way (that I am not seeing)?

Expected e.g., 
```
$ dpkg -I foo.deb
...
 License: BSD-3-Clause
...
```

### PR DESCRIPTION
**Pull request by nstng**
This change adds an optional attribute to `pkg_deb` called `license` for providing a short license description of the packaged code to the artifact meta info. E.g.,

```
pkg_deb(
  ...
  license = "BSD-3-Clause",
  ...
)

$ dpkg -I foo.deb
...
 License: BSD-3-Clause
...
```

Closes #650.

Signed-off-by: Nils Semmelrock <nils.semmelrock@tngtech.com>

### CODE REVIEW

**Comment by nstng** [comment]
First PR here :) I did not find special contributor guidelines (apart from the CLA - which is in progress). Let me know if I missed something.
Unsure about adding this change to the docs - I only see 0.8.0 docs, but no upcoming/0.9.0 docs where I could add this.

**Comment by aiuto** [comment]
CLA https://cla.developers.google.com/

Don't worry about the docs. I do those periodically.  But, if you want to try it, do what is in doc_build/BUILD
```
bazel build //doc_build:reference
cp bazel-bin/doc_build/reference.md docs/latest.md
git commit -m 'update docs' docs/latest.md
```

File: tests/deb/pkg_deb_test.py [hunk 1]
```diff
@@ -158,6 +158,7 @@ def test_description(self):
 	'Replaces: oldpkg',
 	'Breaks: oldbrokenpkg',
 	'Provides: hello',
+  'License: Apache-2.0',
```
Comment (aiuto) [inline tests/deb/pkg_deb_test.py:161]: The indent looks off.

File: docs/pkg_deb.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ pkg_deb_impl(<a href="#pkg_deb_impl-name">name</a>, <a href="#pkg_deb_impl-archi
 | distribution |  TBD   | String | optional | "unstable" |
 | enhances |  See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.   | List of strings | optional | [] |
 | homepage |  The homepage of the project.   | String | optional | "" |
+| license |  The license of the project.   | String | optional | "" |
```
Comment (aiuto) [inline docs/pkg_deb.md:41]: Don't bother. This is not the real doc file. I think this was an accidentally add while fixing the documentation generation.

File: tests/deb/pkg_deb_test.py (hunk 1 above)
Comment (nstng) [reply tests/deb/pkg_deb_test.py]: :+1: there were a couple of tabs in the file - took the liberty and changed them to spaces (looks like spaces are commonly used for indentation).

File: docs/pkg_deb.md (hunk 2 above)
Comment (nstng) [reply docs/pkg_deb.md]: ok, reverted

**Comment by nstng** [comment]
> CLA https://cla.developers.google.com/

In progress - I need to do this via corporate (I don't expect issues, I just have to wait for approval)

> Don't worry about the docs. I do those periodically. But, if you want to try it, do what is in doc_build/BUILD
> 
> ```
> bazel build //doc_build:reference
> cp bazel-bin/doc_build/reference.md docs/latest.md
> git commit -m 'update docs' docs/latest.md
> ```

added second commit with generated docs

**Comment by aiuto** [comment]
Great. I'll wait for the CLA. Ping this PR when you are ready and I can merge.

**Comment by nstng** [comment]
Hi @aiuto, got my CLA finally setup :)
PR is rebased and ready-for-review.

**Review by aiuto — APPROVED** [review]

**Pull request merged**